geom/type_envelope_test.go

```diff
@@ -609,3 +609,19 @@ func TestBoundingDiagonal(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvelopeString(t *testing.T) {
+	for i, tc := range []struct {
+		env  Envelope
+		want string
+	}{
+		{Envelope{}, "ENV EMPTY"},
+		{twoPtEnv(1.5, 2.5, 3.5, 4.5), "ENV(1.5 2.5,3.5 4.5)"},
```

Comment:
Was there a requirement to have a shortened version of the work "envelope"? 

I feel ENV is a common shorthand for "environment", and could be a little confusing. Moreover, WKTs use the full word such as POLYGON or MULTIPOLYGON, and it looks like we're using a similar syntax to WKTs, so using ENVELOPE maintains consistency with how WKTs are named.

What do you think?

Reply:
I hadn't considered that `ENV` could be confused with environment, and I agreed that it definitely could be!

I've updated to `ENVELOPE` in 888d5c6f1427e2b8d3eb86409245b230d98af152.